packages/mdc-side-sheet/README.md

```diff
@@ -1,4 +1,318 @@
+      <div><label><input type="checkbox" checked> Projects</label></div>
+      <div><label><input type="checkbox"> Reminders</label></div>
+      <div><label><input type="checkbox"> Work</label></div>
+    </div>
+  </aside>
+
+  <div class="mdc-side-sheet-scrim"></div>
+
+  <div>Main Content</div>
+</body>
+```
+
+> The `mdc-side-sheet-scrim` next sibling element protects the app's UI from interactions while the side-sheet is open.
+
+## Style Customization
+
+### CSS Classes
+
+Class | Description
+--- | ---
+`mdc-side-sheet` | Mandatory.
+`mdc-side-sheet--dismissible` | Dismissible side-sheet variant class.
+`mdc-side-sheet--modal` | Modal side-sheet variant class.
+`mdc-side-sheet--open` | If present, indicates that the dismissible side-sheet is in the open position.
+`mdc-side-sheet--opening` | Applied while the side-sheet is animating from the closed to the open position.
+`mdc-side-sheet--closing` | Applied while the side-sheet is animating from the open to the closed position.
+`mdc-side-sheet__content` | Scrollable content area of the side-sheet.
+`mdc-side-sheet-app-content` | Dismissible variant only. Sibling element that is resized when the side-sheet opens/closes.
+`mdc-side-sheet-scrim` | Modal variant only. Used for the overlay on the app content.
+
+
+### Sass Mixins
+
+Mixin | Description
+--- | ---
+`mdc-side-sheet-ink-color` | Sets the ink color of text within the side sheet.
+`mdc-side-sheet-surface-fill-color($color)` | Sets the background color of `mdc-side-sheet`.
+`mdc-side-sheet-surface-fill-color-accessible($color)` | Sets the fill color to `$color`, and text ink color to an accessible color relative to `$color`.
+`mdc-side-sheet-scrim-fill-color($color)` | Sets the fill color of `mdc-side-sheet-scrim`.
+`mdc-side-sheet-stroke-color($color)` | Sets border color of `mdc-side-sheet` surface.
+`mdc-side-sheet-shape-radius($radius)` | Sets the rounded shape to side-sheet with given radius size. `$radius` can be single radius or list of 2 radius values for trailing-top and trailing-bottom. Automatically flips the radius values in RTL context.
+`mdc-side-sheet-z-index($value)` | Sets the z index of side-sheet. Side Sheet stays on top of top app bar except for clipped variant of side-sheet.
+`mdc-side-sheet-width($width)` | Sets the width of the side-sheet. In the case of the dismissible variant, also sets margin required for `mdc-side-sheet-app-content`.
+
+## Accessibility
+
+### Focus Management
+
+It is recommended to shift focus to the first focusable element in the main content when side-sheet is closed or one of the destination items is activated. (By default, MDC Side Sheet restores focus to the menu button which opened it.)
+
+#### Dismissible Side Sheet
+
+Restore focus to the first focusable element when a list item is activated or after the side-sheet closes. Do not close the side-sheet upon item activation, since it should be up to the user when to show/hide the dismissible side-sheet.
```

Comment:
This is left over from drawer and needs to be updated or removed. Check if there's particular guidance RE whether anything aside from an optional close affordance should close the side sheet (and I'm not sure a close affordance is required since you can toggle it via the same icon you opened it with?)